### ISSUE TITLE
Replace animated GIFs with videos

### ISSUE THREAD

**Issue by clhenrick** [clhenrick/clhenrick.io#60]
Some blog posts such as [Prototyping with Observable Notebooks](https://clhenrick.io/blog/prototyping-with-observable-notebooks/) contain animated GIFs that cannot be paused or stopped. These files are also fairly big so using a video mp4 format would make them smaller to download.
